src/Share/Survey/ScheduleInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.PowerShell.Share.Survey
+{
+    internal class ScheduleInfo
+    {
+        [JsonProperty(PropertyName = "lastPromptDate")]
+        internal string LastPromptDate { get; set; }
```

Comment:
Predictor may have higher prompt frequency. Can we allow it set its value? If yes, LastPromptDate should be in moduleinfo of predictor.

Reply:
AzPredictor has its own logic at line193